randutil/crypto_test.go

```diff
@@ -45,3 +48,81 @@ func (s *cryptoRandutilSuite) TestCryptoToken(c *C) {
 	c.Assert(err, IsNil)
 	c.Check(b, HasLen, 5)
 }
+
+var (
+	KernelTestUUID = "1031319a-b661-4c01-aafa-6def8a118944"
```

Comment:
(super-nitpick) no need to make this an exported symbol (but then this is inside _test which is never imported anyway so it's a bit moot, this is clearly my ocd talking here)